client/src/styles/entitlements/_views.scss

```diff
@@ -201,3 +201,37 @@
     }
   }
 }
+
+.quipucords-wizard {
+  .wizard-pf-row {
+    height: 400px;
```

Comment:
This likely shouldn't be set in mobile mode.

Reply:
Min-width it is